src/proof/lfsc/lfsc_node_converter.cpp

```diff
@@ -200,10 +200,8 @@ Node LfscNodeConverter::postConvert(Node n)
     {
       return opc;
     }
-    std::vector<Node> children;
-    children.push_back(opc);
-    children.insert(children.end(), n.begin(), n.end());
-    return postConvert(nm->mkNode(APPLY_UF, children));
+    std::vector<Node> children(n.begin(), n.end());
+    return postConvert(mkApplyUf(opc, children));
```

Comment:
```suggestion
    return postConvert(mkApplyUf(opc, std::vector<Node>{n.begin(), n.end}));
```
I think this should work.